### PR TITLE
POC for validation tests using test data

### PR DESCRIPTION
**Pull request by srowlands-moj**

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/CalculationTransactionalServiceTest.kt [hunk 1]
```diff
@@ -157,6 +158,44 @@ class CalculationTransactionalServiceTest {
...
+    }
+    log.info(
+      "Example $exampleType/$exampleNumber outcome BookingCalculation: {}",
+      TestUtil.objectMapper().writeValueAsString(calculatedReleaseDates),
+    )
+    val bookingData = jsonTransformation.loadCalculationResult("$exampleType/$exampleNumber")
+
+    assertEquals(bookingData.dates, calculatedReleaseDates.dates)
+    assertEquals(bookingData.effectiveSentenceLength, calculatedReleaseDates.effectiveSentenceLength)
+    assertThat(returnedValidationMessages).hasSize(1) // etc
```
Comment (srowlands-moj) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/CalculationTransactionalServiceTest.kt:196]: I'm guessing that this didn't run ok? Else you'd be expecting to get an error message I thought your changes to get the longest possible sentences to check date of award etc

Comment (srowlands-moj) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/CalculationTransactionalServiceTest.kt:196]: Unless I'm not getting the test that is

Comment (srowlands-moj) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/CalculationTransactionalServiceTest.kt:196]: As clearly the checks completed ok 👀

**Review by joelstobart-moj — APPROVED** [review]
LGTM

**Review by joelstobart-moj — APPROVED** [review]
LGTM

**Pull request merged**